Tests/test_file_png.py

```diff
@@ -778,6 +778,18 @@ class MyStdOut:
         with Image.open(mystdout) as reloaded:
             assert_image_equal_tofile(reloaded, TEST_PNG_FILE)
 
+    def test_truncated_end_chunk(self):
```

Comment:
```suggestion
    def test_truncated_end_chunk(self) -> None:
```
To follow on with the type hinting added in #7756